cms/envs/test.py

```diff
@@ -122,10 +122,19 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': "edxtest",
+        'NAME': 'edxapp',
```

Comment:
Will it use edx-platform database instead of test database?

Reply:
No. 
Everytime we run untitest a new EC2 instance is launched which has an empty "edxapp" database in it. This is entirely different from edxapp database of dev, staging or other environments.